packages/url_launcher/url_launcher_platform_interface/lib/src/types.dart

```diff
@@ -44,6 +45,11 @@ class InAppWebViewConfiguration {
 
   /// Additional headers to pass in the load request.
   final Map<String, String> headers;
+
+  /// Whether or not to show the webpage title when using Chrome Custom Tabs.
+  ///
+  /// Only works on Android.
+  final bool showTitle;
```

Comment:
Adding this to `InAppWebViewConfiguration` becomes somewhat problematic in light of https://github.com/flutter/packages/pull/5155.

We could potentially overload this to apply to browser views as well, but given that the set of things that work for web views and the set of things that work for browser views is likely to stay completely disjoint, it would probably be better to add a new configuration object for browser view settings instead. (It won't just be this one; SFSafariViewController has a few options, for instance.)

Reply:
Good point! Added `InAppBrowserConfiguration` to address this